experimental/packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -309,4 +326,61 @@ export class NodeSDK {
         .then(() => {})
     );
   }
+
+  private configureLoggerProviderFromEnv(): void {
+    const logExportersList = process.env.OTEL_LOGS_EXPORTER;
+    if (!logExportersList) {
+      diag.info(`No log exporters specified. Logs will not be exported.`);
+      return;
+    }
+
+    const exporters: LogRecordExporter[] = [];
+    const enabledExporters = filterBlanksAndNulls(logExportersList.split(','));
+
+    if (enabledExporters.includes('none')) {
+      diag.info(
+        `OTEL_LOGS_EXPORTER contains "none". Logger provider will not be initialized.`
+      );
+      return;
+    }
+
+    enabledExporters.forEach(exporter => {
+      if (exporter === 'otlp') {
+        const protocol = (
+          process.env.OTEL_EXPORTER_OTLP_LOGS_PROTOCOL ??
+          process.env.OTEL_EXPORTER_OTLP_PROTOCOL
+        )?.trim();
+        switch (protocol) {
+          case 'grpc':
+            exporters.push(new OTLPGrpcLogExporter());
+            break;
+          case 'http/json':
+            exporters.push(new OTLPHttpLogExporter());
+            break;
+          case 'http/protobuf':
+            exporters.push(new OTLPProtoLogExporter());
+            break;
+          default:
+            diag.debug(
+              `Unsupported or undefined OTLP logs protocol. Using http/protobuf.`
```

Comment:
I think the empty case (i.e. the protocol was not specified and the default is being used) and the `default:` case (i.e. the specified protocol is unknown/invalid) should be handled separately. An unknown/invalid envvar value is meant to result in a `diag.warn` and then gracefully use the default.

From https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/configuration/sdk-environment-variables.md#enum-value

> if the user provides a value the implementation does not recognize, the implementation MUST generate a warning and gracefully ignore the setting.